homeassistant/components/advantage_air/services.yaml

```diff
@@ -7,3 +7,9 @@ set_time_to:
     minutes:
       description: Minutes until action
       example: "60"
+set_myzone:
+  description: Change which zone is set as the reference for temperature control
+  fields:
+    entity_id:
```

Comment:
Please use our new service description api:

https://developers.home-assistant.io/docs/dev_101_services#service-descriptions

https://www.home-assistant.io/docs/blueprint/selectors/#target-selector

We don't add the `entity_id` field anymore. Use a target instead. See eg:
https://github.com/home-assistant/core/blob/4f5c7454929f76d6415b418fc2f83fa9862f7395/homeassistant/components/wled/services.yaml#L50-L66

Reply:
This has been implemented. https://github.com/home-assistant/core/pull/46934/files#diff-1b4fc31aa55ab7f95cee85ba410d629ad803da2f92f61859374edf91152b9828